tests/test_mets_server.py

```diff
@@ -32,11 +32,11 @@ def _start_mets_server(*args, **kwargs):
     if exists(WORKSPACE_DIR):
         rmtree(WORKSPACE_DIR, ignore_errors=True)
 
-    copytree(assets.path_to('kant_aufklaerung_1784/data'), WORKSPACE_DIR)
+    copytree(assets.path_to('SBB0000F29300010000/data'), WORKSPACE_DIR)
     workspace = Workspace(Resolver(), WORKSPACE_DIR)
     p = Process(target=_start_mets_server, kwargs={'workspace': workspace, 'url': request.param})
     p.start()
-    sleep(2)  # sleep to start up server
+    sleep(.1)  # sleep to start up server
```

Comment:
I would still keep this high enough - 1 second.

Reply:
Indeed, that was just for testing, corrected in 97a05facf